transformer_engine/pytorch/float8_tensor.py

```diff
@@ -574,7 +552,7 @@ def __torch_dispatch__(cls, func, types, args, kwargs=None):
                 # Directly copy FP8 data if possible
                 if dst._fp8_dtype == src._fp8_dtype:
                     dst._data.copy_(src._data)
-                    dst._scale_inv = src._scale_inv.clone()
+                    dst._scale_inv.copy_(src._scale_inv.detach().clone())
```

Comment:
`clone` is redundant with `copy_`:
```suggestion
                    dst._scale_inv.copy_(src._scale_inv.detach())
```